lib/chef/provider/aws_vpc.rb

```diff
@@ -99,6 +99,31 @@ def destroy_aws_object(vpc)
           end
         end
       end
+
+      #SDK V2
+      vpc_peering_connections = []
```

Comment:
Another testing request I forgot earlier - can you add a test to `aws_vpc_spec.rb` that tests this logic?  Something like...

```
context "foo" do
  aws_vpc "vpc 1"
  aws_vpc "vpc 2"
  aws_vpc_peering_connection "conn" do
    vpc "vpc 1"
    peer_vpc "vpc 2"
  end

  it "deletes the peer connection when deleting the vpc" do
    converge {
      aws_vpc "vpc 1" do
        action :destroy
      end
    }
    # Lookup the peering connections
    expect(peering_connection).to eq(nil)
  end
end
```